lib/src/mock_location.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:math';
+
+import 'constants.dart' show random;
+
+final R = 6372800;
+
+class Location {
+  double lat;
+  double lon;
+  Location(this.lat, this.lon);
+}
+
+/// Generate random latitude longitude in `radius` from `centerLat` to `centerLon`, inclusive.
+/// If any argument is null, random location will be returned.
+Location mockLocation([double centerLat, double centerLon, int radius]) {
```

Comment:
`centerLat` and `centerLon` can be renamed to `latitude` and `longitude` ? I don't see reason to include `center` in front ?